src/VersionTool/ProcessRunner.cs

```diff
@@ -0,0 +1,136 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Text;
+
+namespace VersionTool
+{
+    internal class ProcessRunner
```

Comment:
If it's not super painful -  could you use the `Command` type from `https://www.nuget.org/packages/Microsoft.DotNet.Cli.Utils/1.0.0-preview2-003121`? It's pretty functional and avoids having boiler plate wrapper code for `Process`.